bin/bromley/make-parks-database

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
+
+=head1 NAME
+
+make-parks-database
+
+=head1 USAGE
+
+make-parks-database
+
+=head1 DESCRIPTION
+
+Creates a SQLite database for park name lookups in Bromley.
+
+=cut
+
+use strict;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use DBI;
+use Text::CSV;
+use FixMyStreet;
+use BromleyParks;
+use Cwd qw(abs_path);
+use feature qw(say);
+
+my $parks_csv = FixMyStreet->path_to('data/bromley/parks.csv');
+my $db = DBI->connect('dbi:SQLite:dbname='. BromleyParks::database_file);
+
+say "Creating parks database...";
+
+$db->do(<<EOF) or die;
+CREATE TABLE IF NOT EXISTS parks (
+    name TEXT NOT NULL,
+    ward TEXT NOT NULL,
```

Comment:
I put them in there "just in case", but having given it a bit more thought I don't think it's necessary. Easy enough to recreate the database if we do end up needing those fields. Have removed.